viewer/src/__tests__/datasources/local/pointcloud/createLocalPointCloudModel.test.ts

```diff
@@ -4,10 +4,10 @@
 
 // @ts-ignore
 import * as Potree from '@cognite/potree-core';
-import { createLocalPointCloudModel } from '@/dataModels/pointCloud/public/createLocalPointCloudModel';
+import { createLocalPointCloudModel } from '@/dataModels/point-cloud/public/createLocalPointCloudModel';
```

Comment:
revert to pointCloud, and then let's change the package names ones we have merged the outstanding pr's